src/Uno.Toolkit.RuntimeTests/Tests/TabBarTests.cs

```diff
@@ -261,5 +265,185 @@ public async Task Verify_Indicator_Transitions(Orientation orientation, Indicato
+		[DataRow(false, DisplayName = "Verify Indicator Below")]
+		public async Task Verify_Indicator_Placement(bool isAbove)
+		{
+			var item = new TabBarItem
+			{
+				Content = new Border
+				{
+					Padding = new Thickness(20),
+					Background = new SolidColorBrush(Colors.Green),
+				}
+			};
+
+			var SUT = new TabBar
+			{
+				Background = new SolidColorBrush(Colors.Transparent),
+				SelectionIndicatorContentTemplate = XamlHelper.LoadXaml<DataTemplate>(@"
+					<DataTemplate>
+						<Border Background=""Red"" />
+					</DataTemplate>
+				"),
+				SelectedIndex = 0,
+				SelectionIndicatorPlacement = isAbove ? IndicatorPlacement.Above : IndicatorPlacement.Below,
+			};
+			SUT.Items.Add(item);
+
+			await UnitTestUIContentHelperEx.SetContentAndWait(SUT);
+
+			var belowPresenter = VisualTreeHelperEx
+				.GetFirstDescendant<TabBarSelectionIndicatorPresenter>(SUT, x => x.Name == "BelowSelectionIndicatorPresenter");
+			var abovePresenter = VisualTreeHelperEx
+				.GetFirstDescendant<TabBarSelectionIndicatorPresenter>(SUT, x => x.Name == "AboveSelectionIndicatorPresenter");
+
+			
+			var renderer = await TakeScreenshot(SUT);
+
+			var centerPoint = item.TransformToVisual(SUT).TransformPoint(new Point(item.ActualWidth / 2, item.ActualHeight / 2));
+			var pixel = await GetColorAt(renderer, (int)centerPoint.X, (int)centerPoint.Y);
+			
+
+			if (isAbove)
+			{
+				Assert.AreEqual(Visibility.Collapsed, belowPresenter!.Visibility);
+				Assert.AreEqual(Visibility.Visible, abovePresenter!.Visibility);
+
+				if (pixel is { })
+				{
+					AssertExpectedColor(Colors.Red, pixel);
+				}
+			}
+			else
+			{
+				Assert.AreEqual(Visibility.Visible, belowPresenter!.Visibility);
+				Assert.AreEqual(Visibility.Collapsed, abovePresenter!.Visibility);
+
+				if (pixel is { })
+				{
+					AssertExpectedColor(Colors.Green, pixel);
+				}
+			}
+		}
+
+		private static void AssertExpectedColor(Color expected, Color? actual)
+		{
+			Assert.AreEqual(expected.A, actual?.A);
+			Assert.AreEqual(expected.R, actual?.R);
+			Assert.AreEqual(expected.G, actual?.G);
+			Assert.AreEqual(expected.B, actual?.B);
+		}
+
+		private static async Task<Color?> GetColorAt(RenderTargetBitmap? bitmap, int x, int y)
+		{
+			if (bitmap is null)
+			{
+				return null;
+			}
+
+			var pixelBuffer = await bitmap.GetPixelsAsync();
+			var pixels = pixelBuffer.ToArray();
+
+			var offset = (y * bitmap.PixelWidth + x) * 4;
+			var a = pixels[offset + 3];
+			var r = pixels[offset + 2];
+			var g = pixels[offset + 1];
+			var b = pixels[offset + 0];
+
+			return Color.FromArgb(a, r, g, b);
+		}
+
+
+		private static async Task<RenderTargetBitmap?> TakeScreenshot(UIElement? element)
+		{
+			if (CanTakeScreenshot())
+			{
+				var renderer = new RenderTargetBitmap();
+				await renderer.RenderAsync(element);
+				return renderer;
+			}
+			else
+			{
+				return null;
+			}
+		}
+
+		private static bool CanTakeScreenshot()
```

Comment:
```suggestion
		private static bool IsScreenshotSupport()
```
or just `RuntimeImageAssert::IsSupported()`